frontend/src/components/VideoCarrouselSlick.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import Slider from "react-slick";
+import djangoAPI from "../api/djangoAPI";
+import SampleNextArrow from "./SampleNextArrow";
+import SamplePrevArrow from "./SamplePrevArrow";
+import '../style/style.scss';
+
+
+class VideoCarrouselSlick extends Component {
+
+    CARROUSSEL_SIZE = 5;
+    VIDEO_PER_PAGE = 10;
```

Comment:
Would be great not to hardcode this parameter.
This is a configuration in the backend. If we change it, we have to change it in both places.
So we should find a way that the client gets this parameters from the backend.
Could we for instance get this parameter from the first search request send to the server?